cmd/kube-proxy/app/server.go

```diff
@@ -150,12 +162,32 @@ func AddFlags(options *Options, fs *pflag.FlagSet) {
 	utilfeature.DefaultFeatureGate.AddFlag(fs)
 }
 
+func NewOptions() Options {
+	o := Options{
+		config:      new(componentconfig.KubeProxyConfiguration),
+		healthzPort: 10256,
+	}
+
+	var (
+		groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
+		registry             = registered.NewOrDie("")
+	)
+
+	o.scheme = runtime.NewScheme()
```

Comment:
And we don't need the registry here. Just call componentconfig.AddToScheme and componentconfigv1alpha1.AddToScheme.